network/logical_bridge.go

```diff
@@ -24,17 +27,21 @@ func (c evpnClientImpl) CreateLogicalBridge(ctx context.Context, name string, vl
 	defer closer()
 
 	client := c.getEvpnLogicalBridgeClient(conn)
-	ipVtep, err := parseIPAndPrefix(vtepIP)
-	if err != nil {
-		log.Printf("parseIPAndPrefix: error creating vrf: %s\n", err)
-		return nil, err
+	if vni != nil {
+		if vtepIP != "" {
```

Comment:
we can re-write it to have less nesting levels
```go
if vni != nil && vtepIP != "" {
```
`vni` and `vtepIP` are required together, should we report an error if any of them missed?